### ISSUE TITLE
Screen reader is staying silent on submitting the query to Chatbot in Windows.

### ISSUE THREAD

**Issue by msftedad** [FredrikOseberg/react-chatbot-kit#200]
**Title:** Screen reader is staying silent on submitting the query to Chatbot in Windows.

**Test environment:**
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Edge Version 130.0.2849.27
Screen reader: Narrator, NVDA 2024.4.1 Version (2024.4.1.34550), JAWS 2025.2410.106
URL: [Hello from React-chatbot-kit | React-chatbot-kit](https://fredrikoseberg.github.io/react-chatbot-kit-docs/) 

**User Impact:**
Screen reader dependent users will find it difficult to understand whether their action is performed or not if screen reader doesn't announce any information.

**Observation:**
This issue is observed with all three screen readers.

**Pre-Requisite:**
Turn on Screen Readers:
Narrator: Win + Shift + Enter
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Verbosity:
Narrator: Default
JAWS: Beginner, Highest
NVDA: Default

**Repro steps:**

1. Open the URL: [Hello from React-chatbot-kit | React-chatbot-kit](https://fredrikoseberg.github.io/react-chatbot-kit-docs/) in the latest edge browser. React chat bot kit page will open.
2. Press tab key to move to the 'Write your message here' edit field in the 'Conversation with Docs Bot' dialog.
3. Enter any query in the edit field.
4. Press tab key to move to the submit button and activate it using enter key.
5. Verify whether screen reader is announcing any information or not once the response is received from chat bot.

**Actual result:**
On pressing enter key to activate submit button, Screen reader stays silent, and it does not announce any generated response on submitting the query to Chatbot in Windows.

<img width="959" alt="Screen reader is staying silent on submitting the query to Chatbot" src="https://github.com/user-attachments/assets/edbd5256-1833-4872-861f-e51184e917be">

https://github.com/user-attachments/assets/062bfd5b-ba5e-472c-a718-595a919c9e91

**Expected result:**
On pressing enter key to activate submit button, Screen reader should narrate the generated response information, on submitting the query to chatbot in Windows. Ex: when we enter the text as 'Widgets' screen reader should announce as 'Widgets are custom components that you want to render with a chatbot response'.
